front/app/containers/Admin/initiatives/index.tsx

```diff
@@ -68,32 +46,35 @@ const InitiativesPage = memo<WrappedComponentProps & WithRouterProps>(
       setTabs(insertConfiguration<ITab>(data));
 
     const { pathname } = location;
+
     return (
       <>
         <Outlet
           id="app.containers.Admin.initiatives.tabs"
           onData={handleData}
           formatMessage={formatMessage}
         />
-        <TopContainer>
-          <ActionsContainer>
-            <Button
-              id="e2e-new-proposal"
-              buttonStyle="cl-blue"
-              icon="initiatives"
-              linkTo={`/initiatives/new`}
-              text={formatMessage(messages.addNewProposal)}
-              onClick={onNewProposal(pathname)}
-            />
-          </ActionsContainer>
-        </TopContainer>
-        <TabbedResource resource={resource} tabs={tabs}>
+        <TabbedResource
+          resource={{
+            title: formatMessage(messages.titleInitiatives),
+            rightSideCTA: (
+              <Button
+                id="e2e-new-proposal"
+                buttonStyle="cl-blue"
+                icon="initiatives"
+                linkTo={`/initiatives/new`}
+                text={formatMessage(messages.addNewProposal)}
+                onClick={onNewProposal(pathname)}
+              />
+            ),
+          }}
+          tabs={tabs}
+        >
           <HelmetIntl
```

Comment:
Clean up of the UI: the add proposal button is now on the same line as the title.